apps/test/unit/templates/certificates/CongratsTest.js

```diff
@@ -23,45 +19,42 @@ describe('Congrats', () => {
       );
       expect(wrapper.find(Certificate).exists()).to.be.true;
     });
+  });
 
-    it(`renders a StudentsBeyondHoc for user type ${userType} for HOC course`, () => {
-      const wrapper = shallow(
-        <Congrats {...defaultProps} userType={userType} isHocTutorial />
-      );
-      expect(wrapper.find(StudentsBeyondHoc).exists()).to.be.true;
-    });
+  it('renders curriculum catalog button, for teachers', () => {
+    const wrapper = shallow(<Congrats {...defaultProps} userType="teacher" />);
 
-    it(`renders a GraduateToNextLevel for user type ${userType} for CSF course`, () => {
-      const wrapper = shallow(
-        <Congrats {...defaultProps} userType={userType} isHocTutorial={false} />
-      );
-      expect(wrapper.find(GraduateToNextLevel).exists()).to.be.true;
-    });
+    wrapper.find('a[href="/catalog"]');
```

Comment:
Can you wrap this in `expect(wrapper ... ]').exists()).to.be.true;` statements like the other tests please